apps/web/pages/apps/index.tsx

```diff
@@ -52,7 +52,11 @@ export default function Apps({
           <TrendingAppsSlider items={appStore} />
         </>
       )}
-      <AllApps apps={appStore} searchText={searchText} />
+      <AllApps
+        apps={appStore}
+        searchText={searchText}
+        categories={categories.map((category: string) => category.name)}
```

Comment:
Pass categories down to the all apps component since we already get this value server side.